src/main/java/com/amazon/opendistroforelasticsearch/knn/index/codec/KNN80Codec/KNN80Codec.java

```diff
@@ -50,11 +43,11 @@
     private final CompoundFormat compoundFormat;
     private Codec lucene80Codec;
 
-    public static final String CODEC_NAME = "KNN80Codec";
-    public static final String LUCENE_CODEC = "Lucene80";
+    public static final String KNN_80 = "KNN80";
```

Comment:
We should  not be changing codec name. This can result in compatibility issues when migrated from ODFE1.4 to 1.6 as segments would be created with KNN80Codec and it would not be able to locate it.

Reply:
Makes sense. Will Revert everything back to KNNXXCodec naming convention